tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1349,6 +1349,341 @@ def func_scalar(self, xp, a, scalar):
         a /= scalar
 
 
+_bwise_in_out_dtypes = dtype_utils._permutate_dtype_mapping([
+    (('bool_', 'int8'), 'int8'),
+    (('bool_', 'int16'), 'int16'),
+    (('bool_', 'int32'), 'int32'),
+    (('bool_', 'int64'), 'int64'),
+    (('bool_', 'uint8'), 'uint8'),
+    (('int8', 'int16'), 'int16'),
+    (('int8', 'int32'), 'int32'),
+    (('int8', 'int64'), 'int64'),
+    (('int16', 'int32'), 'int32'),
+    (('int16', 'int64'), 'int64'),
+    (('int32', 'int64'), 'int64'),
+])
+
+_bwise_inplace_invalid = [
+    (('bool_', 'int8'), 'int8'),
+    (('bool_', 'int16'), 'int16'),
+    (('bool_', 'int32'), 'int32'),
+    (('bool_', 'int64'), 'int64'),
+    (('bool_', 'uint8'), 'uint8'),
+]
+
+_bwise_in_out_inplace_dtypes = [
+    dtypes for dtypes in _bwise_in_out_dtypes
+    if dtypes not in _bwise_inplace_invalid
+]
+
+_bwise_scalar_invalid = [
+    (('float16',), int, 'float16'),
+    (('float32',), int, 'float32'),
+    (('float64',), int, 'float64'),
+    (('float64',), numpy.int8, 'float64'),
+    (('float16',), numpy.int64, 'float16'),
+]
+
+_bwise_in_out_scalar_dtypes = [
+    dtypes for dtypes in _in_out_dtypes_array_int_scalar
+    if dtypes not in _bwise_scalar_invalid
+]
+
+_bitwise_params = (
```

Comment:
Could you merge master and check if broadcast works well?

Reply:
If I am correct broadcasting is already checked using,

https://github.com/chainer/chainer/blob/7ee1103ffcc002aa45749befc2a39f6f99e3345f/tests/chainerx_tests/unit_tests/routines_tests/test_math.py#L295-L329

Reply:
Sorry, I overlooked.